site/_docs/reference.md

```diff
@@ -2752,6 +2752,8 @@ In the following:
 | * | ATANH(numeric)                                 | Returns the inverse hyperbolic tangent of *numeric*
 | f | BITAND_AGG(value)                              | Equivalent to `BIT_AND(value)`
 | f | BITOR_AGG(value)                               | Equivalent to `BIT_OR(value)`
+| * | BITCOUNT(integer)                              | Returns the bitwise COUNT of non-null *integer*
+| m | BIT_COUNT(numeric)                             | Returns the bitwise COUNT of the integer portion of non-null *numeric*
```

Comment:
do these functions apply to binary, varbinary , decimal, float? if so, say so

Reply:
Updated the documentation to mention that it also supports binary and numeric (in the case of `BIT_COUNT()`).